tests/integration_tests/data_steward/cdr_cleaner/cleaning_rules/backfill_pmi_skip_codes_test.py

```diff
@@ -0,0 +1,245 @@
+                    (32, 3, 40766240, parser.parse('2020-03-02').date(), None,
+                     45905771, None, 'value_as_string', 99, 99, 99, 99, 99, 99,
+                     'Insurance_HealthInsurance', 1585386, 'unit_source_value',
+                     'qualifier_source_value', 99, 'value_source_value', 99),
+                    (41, 4, 3005917, parser.parse('2020-01-01').date(), None,
+                     45905771, None, 'value_as_string', 99, 99, 99, 99, 99, 99,
+                     'TheBasics_Birthplace', 1586135, 'unit_source_value',
+                     'qualifier_source_value', 99, 'value_source_value', 99),
+                    (1000000000005, 2, 40766240,
+                     parser.parse('2020-02-01').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'Insurance_HealthInsurance', 1585386, None, None, 903096,
+                     'PMI_Skip', None),
+                    (1000000000006, 2, 40767407,
+                     parser.parse('2020-02-01').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'OverallHealth_MenstrualStopped', 1585784, None, None,
+                     903096, 'PMI_Skip', None),
+                    (1000000000008, 2, 1586166,
+                     parser.parse('2020-02-01').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'ElectronicSmoking_ElectricSmokeParticipant', 1586166,
+                     None, None, 903096, 'PMI_Skip', None),
+                    (1000000000010, 3, 40767407,
+                     parser.parse('2020-03-02').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'OverallHealth_MenstrualStopped', 1585784, None, None,
+                     903096, 'PMI_Skip', None),
+                    (1000000000012, 3, 1586166,
+                     parser.parse('2020-03-02').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'ElectronicSmoking_ElectricSmokeParticipant', 1586166,
+                     None, None, 903096, 'PMI_Skip', None),
+                    (1000000000013, 4, 40766240,
+                     parser.parse('2020-01-01').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'Insurance_HealthInsurance', 1585386, None, None, 903096,
+                     'PMI_Skip', None),
+                    (1000000000015, 4, 1586166,
+                     parser.parse('2020-01-01').date(), None, 45905771, None,
+                     None, 903096, 0, 0, None, None, None,
+                     'ElectronicSmoking_ElectricSmokeParticipant', 1586166,
+                     None, None, 903096, 'PMI_Skip', None),
+                ]
+            },
+            {
+                'fq_table_name':
```

Comment:
@hiro-mishima the above test was very comprehensive. But this test is not needed as we are not modifying anything in-person table. I mean it is not going to help us test anything and increase test running time. Is there anything I am missing regarding this?